sys-kernel/coreos-kernel/coreos-kernel-4.15.0.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 2014-2016 CoreOS, Inc.
```

Comment:
Should we bump the year in new files?

Reply:
They're copies of other files and haven't really changed at all, so in this case I'd say no.